container/sagemaker/python_service.py

```diff
@@ -206,6 +216,7 @@ def add_routes(self, application):
         if self._enable_model_manager:
             model_manager_resource = ModelManagerResource()
             application.add_route('/models', model_manager_resource)
+            application.add_route('/models/{model_name}', model_manager_resource)
```

Comment:
not clear what this has to do with deleting a model

Reply:
The deleting model request is 'DELETE /models/{model_name}', we need to pass the model_name into the `on_delete` function. With only '/models', falcon app cannot recognize `/models/{model_name}` request uri.

Reply:
ok